woocommerce/wc-admin
# Update/5348 trigger ces on changing setting

Fixes #5348 

This PR triggers the CES survey when settings get updated.

- Added a base unit test class for the upcoming changes.


### Screenshots
![Screen Shot 2020-11-09 at 10 21 19 AMth_](https://user-images.githubusercontent.com/4723145/98580745-57884b00-2275-11eb-83db-e455a4ad4513.jpg)

### Detailed test instructions:

1. Navigate to WooCommerce -> Settings
2. Change a value
3. Click [ Save Changes ]
4. You should see the snackbar at the bottom left corner on the next page.

To repeat the test, delete `woocommerce_ces_shown_for_actions` and `woocommerce_ces_tracks_queue` from the `wp_options` table.
